stylefunction.js

```diff
@@ -601,6 +601,19 @@ export default function(olLayer, glStyle, source, resolutions, spriteData, sprit
             label = label.toLowerCase();
           }
           const wrappedLabel = type == 2 ? label : wrapText(label, font, getValue(layer, 'layout', 'text-max-width', zoom, f));
+          const letterSpacing = getValue(layer, 'layout', 'text-letter-spacing', zoom, f);
+          if (letterSpacing && letterSpacing > 0.01 /* default is 0.01 */ ) {
+            let wrappedLabelWithLetterSpacing = '';
+            wrappedLabel.split('\n').forEach((line) => {
+              let joinSpaceString = "",
```

Comment:
Please rebase your pull request on top of master so the CI catches this, and use single quotes.